source/docs/software/dashboards/dashboard-intro.rst

```diff
@@ -0,0 +1,42 @@
+Choosing a Dashboard
+====================
+
+A dashboard is a program used to retrieve and display information about the operation of your robot.  There are two main types of dashboards that teams may need: driver and programmer dashboards.  Some dashboards will try to accommodate both purposes.
+
+Driver Dashboard
+----------------
+
+During competition the drive team will use this dashboard to get information from the robot.  It should focus on conveying key information instantly.  This is often best accomplished by using large, colorful, and easy to understand visual elements.
+
+Most teams will use this dashboard to select their autonomous routine.  It is often helpful to hide this choice after the selection is made as that information is not needed during the match.
+
+Take caution to carefully consider what *needs* to be on this dashboard and if there is another better way of communicating that information.  Any members of the drive team (especially the driver) looking at the dashboard takes their focus away from the match.  Using :ref:`LEDs <docs/software/hardware-apis/misc/addressable-leds:Addressable LEDs>` to indicate the state of your robot is a good example of a way to communicate useful information to the driver without having to take their eyes off the robot.
+
+Programming Dashboard
+---------------------
+
+This dashboard is designed for debugging code and analyzing data from the robot. It supports the monitoring of a wide variety of information simultaneously, prioritizing function and utility over simplicity or ease of use. This functionality often includes complex data visualization and graphing across extended periods. In scenarios where there is an overwhelming amount of data to review, real-time analysis becomes challenging. The capability to examine past data and replay it proves to be extremely beneficial. While some dashboards may log data transmitted to them, :ref:`on-robot telemetry <docs/software/telemetry/datalog:On-Robot Telemetry Recording Into Data Logs>` using the ``Datalog`` class simplifies the process.
+
+Specific Dashboards (oldest to newest)
+--------------------------------------
+
+.. note:: SmartDashboard and Shuffleboard have a long history of aiding FRC teams. However, they are not currently receiving active updates or improvements. Notably, Shuffleboard may experience performance issues on some machines under certain scenarios.
```

Comment:
I think I'd say something like Smatdashboard and Shuffleboard do no have a current WPILib maintainer, so no new features will be added or invasive bugs fixed. PRs from external contributors will be reviewed.